Kickstarter-iOS/Features/ProjectPage/Views/Cells/ReportProjectCell.swift

```diff
@@ -1,46 +1,59 @@
 import Library
+import Prelude
 import SwiftUI
 import UIKit
 
 internal final class ReportProjectCell: UITableViewCell, ValueCell {
-  let isIpad = AppEnvironment.current.device.userInterfaceIdiom == .pad
-
   internal func configureWith(value projectFlagged: Bool) {
+    self.setupTableViewCellStyle(projectFlagged: projectFlagged)
     self.setupReportProjectLabelView(projectFlagged: projectFlagged)
+  }
+
+  internal override func layoutSubviews() {
+    super.layoutSubviews()
+  }
+
+  internal override func bindStyles() {
+    super.bindStyles()
 
-    self.selectionStyle = .none
     self.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: .greatestFiniteMagnitude)
     self.setNeedsLayout()
   }
 
-  internal override func layoutSubviews() {
-    super.layoutSubviews()
+  // MARK: - Private Methods
+
+  private func setupTableViewCellStyle(projectFlagged: Bool) {
+    let accessibilityTraits = projectFlagged
+      ? UIAccessibilityTraits.staticText
+      : UIAccessibilityTraits.button
+
+    _ = self
+      |> baseTableViewCellStyle()
+      |> ReportProjectCell.lens.accessibilityTraits .~ accessibilityTraits
```

Comment:
Thanks for adding this! Unfortunately, I don't think it does anything currently; we'd need to also set `isAccessibilityElement = true` and the accessibility label. I don't see an easy way to set the label based on the swiftui child view. So I'd suggest either leaving this alone for now or adding it just for the button state. Ex:

```
if projectFlagged  {
  self.accessibilityTraits = super.accessibilityTraits
  self.isAccessibilityElement = false
} else {
  self.accessibilityTraits.insert(.button)
  self.isAccessibilityElement = true
  self.accessibilityLabel = Strings.Report_this_project_to()
}
```

Reply:
oh, good callout! I'll look into this in the next PR